uiplib/gui/settingsTab.py

```diff
@@ -104,26 +121,27 @@ def toggle_subreddit(appObj, mainFrame):
             appObj.sub_entry = Text(mainFrame, height=10, width=20)
             for subreddit in appObj.sites_list['reddit']:
                 appObj.sub_entry.insert(INSERT, subreddit + " \n")
-            appObj.sub_label.grid(row=1, column=2, padx=0, pady=0)
-            appObj.sub_entry.grid(row=2, column=2, padx=10, pady=10, sticky=W)
+            appObj.sub_label.grid(row=2, column=2, padx=0, pady=0)
+            appObj.sub_entry.grid(row=3, column=2, padx=10, pady=10, sticky=W)
     else:
         appObj.sub_label.grid_remove()
         appObj.sub_entry.grid_remove()
 
 
-def file_open_helper(appObj):
-    """Open the dialog to choose where pictures are to be saved."""
+def file_open_helper(appObj, folder):
```

Comment:
Actually I suggested that change, We are passing 2 different folders in 2 different calls.
Thats why instead of making 2 function, got it done in one.